src/common/configureStore.js

```diff
@@ -24,7 +24,11 @@ export default function configureStore(options) {
   const store = createStore(
     reducer,
     initialState,
-    applyMiddleware(...middleware)
+    compose(
+      applyMiddleware(...middleware),
+      typeof window === 'object' && typeof window.devToolsExtension !== 'undefined' ?
+        window.devToolsExtension() : f => f
```

Comment:
What 'f' is? Remember, never use single letters for variables. Can you fix it please? Than you